ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/results/tracing/vm/Store.java

```diff
@@ -32,4 +32,8 @@ public String getKey() {
   public String getVal() {
     return val;
   }
+
+  private static String handle0x(final String input) {
+    return input.equals("0x") ? "0x0" : input;
```

Comment:
Rather than patching up inconsistant hex notation, would it be better to address it in the source?  Are we generating bad results?  Is this input filtering?  IMHO standard APIs should have one mapping to each value, and allowing multiple mappings (with 0x, without 0x) is a common source of bugs.

Reply:
`Bytes.toShortHexString` returns `0x` for a byte array containing only zeros. However this API output requires a `0x0` instead. So what do you suggest ? Not using this method ?

Reply:
I put a TODO in the code with a link to Tuweni bug.
https://issues.apache.org/jira/browse/TUWENI-31